analyses/compare-gistic/util/GISTIC-comparison-functions.R

```diff
@@ -11,68 +11,55 @@
 #
 # source(file.path("util", "GISTIC-comparison-functions.R"))
 
-
+genes_file = lgat_amp_genes_file
 #### Implemented in both `01-GISTIC-cohort-vs-histology.Rmd` and `02-GISTIC-tidy-data-prep.Rmd`
 format_gistic_genes <- function(genes_file,
-                                include_peak_info = FALSE) {
+                                include_peak_info = TRUE) {
```

Comment:
Perhaps we should rename this `genes_only` so it's a little more clear one what it's doing.

Reply:
If you do that, then you can make the if statement below just `if (genes_only){` I think that logic notation is a bit easier to folow.